README.md

```diff
@@ -1,5 +1,5 @@
-# nGraph Compiler Stack (Beta)
-
+![](doc/sphinx/source/graphics/ngraph_header.png)
```

Comment:
Would be good to add alt text in the `[]` something like:
`![nGraph Compiler Stack](doc/sphinx/source/graphics/ngraph_header.png)`